alpaca_trade_api/polygon/streamconn.py

```diff
@@ -121,13 +121,12 @@ def __init__(self, key_id=None):
                 await self.connect()
                 if self._streams:
                     await self.subscribe(self._streams)
-
                 break
-            except (ConnectionRefusedError, ConnectionError) as e:
+            except Exception as e:
```

Comment:
I had to make this less specific because in some cases, such as wifi being disabled on the machine running a script, unusual errors are the ones which reach this code.